lib/experimental/fonts/font-library/class-wp-rest-font-faces-controller.php

```diff
@@ -0,0 +1,762 @@
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
+	 */
+	public function delete_item( $request ) {
+		$force = isset( $request['force'] ) ? (bool) $request['force'] : false;
+
+		// We don't support trashing for revisions.
+		if ( ! $force ) {
+			return new WP_Error(
+				'rest_trash_not_supported',
+				/* translators: %s: force=true */
+				sprintf( __( "Font faces do not support trashing. Set '%s' to delete.", 'gutenberg' ), 'force=true' ),
+				array( 'status' => 501 )
+			);
+		}
+
+		return parent::delete_item( $request );
+	}
+
+	/**
+	 * Prepares a single font face output for response.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param WP_Post         $item    Post object.
+	 * @param WP_REST_Request $request Request object.
+	 * @return WP_REST_Response Response object.
+	 */
+	public function prepare_item_for_response( $item, $request ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable -- required by parent class
+		$data = array();
+
+		$data['id']                 = $item->ID;
+		$data['theme_json_version'] = 2;
+		$data['parent']             = $item->post_parent;
+		$data['font_face_settings'] = $this->get_settings_from_post( $item );
+
+		$response = rest_ensure_response( $data );
+		$links    = $this->prepare_links( $item );
+		$response->add_links( $links );
+
+		return $response;
+	}
+
+	/**
+	 * Retrieves the post's schema, conforming to JSON Schema.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @return array Item schema data.
+	 */
+	public function get_item_schema() {
+		if ( $this->schema ) {
+			return $this->add_additional_fields_schema( $this->schema );
+		}
+
+		$schema = array(
+			'$schema'    => 'http://json-schema.org/draft-04/schema#',
+			'title'      => $this->post_type,
+			'type'       => 'object',
+			// Base properties for every Post.
+			'properties' => array(
+				'id'                 => array(
+					'description' => __( 'Unique identifier for the post.', 'default' ),
+					'type'        => 'integer',
+					'readonly'    => true,
+				),
+				'theme_json_version' => array(
+					'description' => __( 'Version of the theme.json schema used for the typography settings.', 'gutenberg' ),
+					'type'        => 'integer',
+					'default'     => 2,
+					'minimum'     => 2,
+					'maximum'     => 2,
+				),
+				'parent'             => array(
+					'description' => __( 'The ID for the parent font family of the font face.', 'gutenberg' ),
+					'type'        => 'integer',
+				),
+				// Font face settings come directly from theme.json schema
+				// See https://schemas.wp.org/trunk/theme.json
+				'font_face_settings' => array(
+					'description'          => __( 'font-face declaration in theme.json format.', 'gutenberg' ),
+					'type'                 => 'object',
+					'properties'           => array(
+						'fontFamily'            => array(
+							'description' => 'CSS font-family value.',
```

Comment:
```suggestion
							'description' => __('CSS font-family value.'),
```

All descriptions should be translatable.

Reply:
Thanks for catching this! I copied this section directly from the theme.json schema and forgot to add translation functions for the description strings.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/57688/commits/32689e176c2d2df55803a1fa06903ac148e8df7d